SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -938,6 +938,21 @@ void CNEMOEulerSolver::Source_Residual(CGeometry *geometry, CSolver **solver_con
         /*--- If necessary, set variables needed for viscous computation ---*/
         if (viscous) {
 
+          // TODO: NOTE: Some of these are set above. They need to be set here as well, otherwise they are passed in incorrectly.
+          // Do not remove.
+          
+          /*--- Set volume ---*/
+          numerics->SetVolume(geometry->nodes->GetVolume(iPoint));
```

Comment:
I think this is fine.  We might need to look into a way to avoid repeating "set" functions in the future.